app/views/user/_posts.html.erb

```diff
@@ -0,0 +1,13 @@
+<% posts = [] %>
+<% current_user.posts.each do |post| %>
```

Comment:
- [ ] have in mind that the user will have to be able to see his friends posts as well, not only his own posts